COPYRIGHT.txt

```diff
@@ -366,6 +366,11 @@ Copyright: 2011, Khaled Mamou
   2003-2009, Erwin Coumans
 License: BSD-3-clause
 
+Files: ./thirdparty/wslay/
+Comment: wslay
```

Comment:
Upstream seems to capitalize it "Wslay".

Reply:
fixed